content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst

```diff
@@ -0,0 +1,233 @@
+
+For consumable products, transfers can be created, but exact quantities at each storage location are
+not tracked.
+
+.. _inventory/product_management/transfer-serv:
+
+Service products cannot be included in transfers, but these products can be `linked to projects and
+tasks for deadline tracking <https://www.youtube.com/watch?v=fix2LGkv13c>`_.
+
+.. _inventory/product_management/po:
+
+Create purchase order
+---------------------
+
+Both storable and consumable products can be included in a request for quotation in the *Purchase*
+app.
+
+However, when receiving consumable products, their on-hand quantity will not change upon validating
+the receipt (e.g. `WH/IN`).
+
+.. _inventory/product_management/manufacture:
+
+Manufacture or subcontract
+--------------------------
+
+Storable and consumable products can both be manufactured, subcontracted, or included in a bill of
+materials (BoM).
+
+.. figure:: type/manufacture.png
+   :align: center
+   :alt: Show "Bill of Materials" and "Used In" smart buttons.
+
+   When the **Bill of Materials** and **Used In** smart buttons are visible on the product form,
+   this indicates the product can be manufactured or used as a component of a |BOM|.
+
+.. _inventory/product_management/package:
+
+Packages
+--------
+
+Both storable and consumable products can be placed in packages.
+
+.. seealso::
+   :doc:`package`
+
+However, for consumable products, the quantity is not tracked, and the product is not listed in the
+package's :guilabel:`Contents`, accessed by going to :menuselection:`Inventory app --> Products -->
+Packages` and selecting the desired package.
+
+.. figure:: type/package-content.png
+   :align: center
+   :alt: Show Packages page, containing the package contents list.
+
+   A consumable product was placed in the package, but the **Content** list does not list it.
+
+If the *Move Entire Package* feature is enabled, moving a package updates the location of the
+contained storable products. However, consumable products' locations are not updated.
+
+.. _inventory/product_management/report:
+
+Inventory report
+----------------
+
+**Only** storable products appear on the following reports.
+
+The *stock report* is a comprehensive list of all on-hand, unreserved, incoming, and outgoing
+storable products. The report is only available to users with :doc:`administrator access
+<../../../../general/users/access_rights>`, and is found by navigating to :menuselection:`Inventory
+app --> Reporting --> Stock`.
+
+.. image:: type/stock-report.png
+   :align: center
+   :alt: Show stock reporting list found in Inventory > Reporting > Stock.
+
+The *location report* is a breakdown of each location (internal, external, or virtual) and the
+on-hand and reserved quantity of each storable product. The report is only available to users with
+:doc:`administrator access <../../../../general/users/access_rights>`, and is found by navigating to
+:menuselection:`Inventory app --> Reporting --> Locations`.
```

Comment:
correct me if I'm wrong, but doesn't 'Storage Locations' need to be activated as well in order to see this report?

Reply:
Great catch!!
```suggestion
The *location report* is a breakdown of each location (internal, external, or virtual) and the
on-hand and reserved quantity of each storable product. The report is only available with the *Storage Location* feature activated, and to users with
:doc:`administrator access <../../../../general/users/access_rights>`. Navigate to the location report by going to
:menuselection:`Inventory app --> Reporting --> Locations`.
```